docs/core/utils.md

```diff
@@ -154,13 +154,27 @@ Checks if device is a smartphone. Returns a `boolean`.
 
 ## Function Utils
 
-### `AFRAME.utils.throttle (function, interval [, optionalContext])`
+### `AFRAME.utils.throttle (function, minimumInterval [, optionalContext])`
 
 Returns a throttled function that is called at most once every
 `minimumInterval` milliseconds. A context such as `this` can be provided to
 handle function binding for convenience.
 
-### `AFRAME.utils.throttleTick (function (t, dt) {...}, interval [, optionalContext])`
+
+```js
+AFRAME.registerComponent('foo', {
```

Comment:
`throttle` should be used on non-tick functions. `throttleTick` should be used for `tick` functions.